include/swift/AST/DiagnosticsParse.def

```diff
@@ -1356,6 +1356,11 @@ ERROR(expected_type_after_as,none,
 ERROR(string_interpolation_extra,none,
       "extra tokens after interpolated string expression", ())
 
+// String interpolation isnt closed by a )
+// ie: `let west = "ye \("`
+ERROR(string_interpolation_unclosed, none,
+      "cannot find ')' to match this opening '(' in string interpolation", ())
```

Comment:
Nit: since this diagnostic is going to point to the opening paren here is no reason for it to use `this` in the text.

Reply:
Resolved